zetaclient/app_context/app_context.go

```diff
@@ -0,0 +1,49 @@
+package appcontext
+
+import (
+	"github.com/rs/zerolog"
+	"github.com/zeta-chain/zetacore/common"
+	"github.com/zeta-chain/zetacore/zetaclient/config"
+	corecontext "github.com/zeta-chain/zetacore/zetaclient/core_context"
+)
+
+type AppContext struct {
+	coreContext  *corecontext.ZeraCoreContext
+	config       *config.Config
+	masterLogger zerolog.Logger
+}
+
+func NewAppContext(
+	coreContext *corecontext.ZeraCoreContext,
+	config *config.Config,
```

Comment:
Do we need a pointer for all the structure? I feel like it could be simplified by being removed in some places.
If it's because it's part of the previous design and changing it here is out of scope, maybe we can create an issue to remove these